src/components/mermaidMap/ProjectSitesMap/ProjectSitesMap.js

```diff
@@ -85,10 +85,11 @@ const ProjectSitesMap = ({ sitesForMapMarkers, choices }) => {
 
     map.current.on('click', 'mapMarkers', (e) => {
       const popupNode = document.createElement('div')
+      const reactRoot = createRoot(popupNode)
       const coordinates = e.features[0].geometry.coordinates.slice()
       const markerProperty = e.features[0].properties
 
-      ReactDOM.render(<Popup properties={markerProperty} choices={choices} />, popupNode)
+      reactRoot.render(<Popup properties={markerProperty} choices={choices} />, popupNode)
```

Comment:
This is a weird pattern using react `render` for a popup, but I didnt want to rearchitect it if I could avoid it... and maybe its there for a reason.